src/apps/common/ns_turn_utils.c

```diff
@@ -157,6 +163,8 @@ static char* str_fac[]={"LOG_AUTH","LOG_CRON","LOG_DAEMON",
 			"LOG_AUTHPRIV","LOG_SYSLOG",
 			0};
 
+#if defined(__unix__) || defined(unix) || defined(__APPLE__) \
```

Comment:
This does not need to be ifdef-ed
Though this is syslog specific code - it does not use any syslog specific APIs

Reply:
If don't use it. then build fail in windows.